app.go

```diff
@@ -932,6 +956,9 @@
 	// Convert raw http response to *http.Response
 	res, err := http.ReadResponse(buffer, req)
 	if err != nil {
+		if errors.Is(err, io.ErrUnexpectedEOF) {
+			return nil, fmt.Errorf("test: got empty response")
+		}
```

Comment:
I can make this change, I was only unsure if it would not match with other errors in the same function created with `fmt.Errorf()`. I think the linter is saying this because there's no need for the formatting ability of `fmt.Errorf()`.